client/ayon_houdini/plugins/load/load_filepath.py

```diff
@@ -50,64 +25,54 @@ class FilePathLoader(plugin.HoudiniLoader):
 
     def load(self, context, name=None, namespace=None, data=None):
 
-        # Get the root node
-        obj = hou.node("/obj")
-
         # Define node name
         namespace = namespace if namespace else context["folder"]["name"]
         node_name = "{}_{}".format(namespace, name) if namespace else name
 
-        # Create a null node
-        container = obj.createNode("null", node_name=node_name)
-
-        # Destroy any children
-        for node in container.children():
-            node.destroy()
-
-        # Add filepath attribute, set value as default value
-        filepath = self.filepath_from_context(context)
-        parm_template_group = container.parmTemplateGroup()
-        attr_folder = hou.FolderParmTemplate("attributes_folder", "Attributes")
-        parm = hou.StringParmTemplate(name="filepath",
-                                      label="Filepath",
-                                      num_components=1,
-                                      default_value=(filepath,))
-        attr_folder.addParmTemplate(parm)
-        parm_template_group.append(attr_folder)
-
-        # Hide some default labels
-        for folder_label in ["Transform", "Render", "Misc", "Redshift OBJ"]:
-            folder = parm_template_group.findFolder(folder_label)
-            if not folder:
-                continue
-            parm_template_group.hideFolder(folder_label, True)
-
-        container.setParmTemplateGroup(parm_template_group)
-
-        container.setDisplayFlag(False)
-        container.setSelectableInViewport(False)
-        container.useXray(False)
-
-        nodes = [container]
-
-        self[:] = nodes
-
-        return pipeline.containerise(
-            node_name,
-            namespace,
-            nodes,
-            context,
-            self.__class__.__name__,
-            suffix="",
-        )
+        # Create node
+        parent_node = get_or_create_avalon_container()
+        node = parent_node.createNode("ayon::generic_loader",
+                                      node_name=node_name)
+        node.moveToGoodPosition()
+
+        # Set representation id
+        parm = node.parm("representation")
+        parm.set(context["representation"]["id"])
+        parm.pressButton()  # trigger callbacks
+
+        # Show Parameter Editor for the Node
+        show_node_parmeditor(node)
```

Comment:
I'd say - remove this. We want this dialog only on the right click context menu from a parameter. Why?
Because if you do it via the loader - the product you wanted to load is already preconfigured.

You might say "But if I load it, it's in the avalon containers group and hard to find, etc.". Yes. But that's a separate issue not needing to be solved here: https://github.com/ynput/ayon-houdini/issues/13
```suggestion
```